utilities/datasource.ts

```diff
@@ -184,7 +194,17 @@ export const marginVerticals = {
 };
 
 export const marginTops = {
-  none: '',
+  none: 'mt-0',
+  '01em': 'mt-01em',
+  '02em': 'mt-02em',
+  '03em': 'mt-03em',
+  '04em': 'mt-04em',
+  '05em': 'mt-05em',
+  '06em': 'mt-06em',
+  '07em': 'mt-07em',
+  '08em': 'mt-08em',
+  '09em': 'mt-09em',
+  '1em': 'mt-1em',
```

Comment:
Added these smaller margin value options for the Typography element to provide more flexibility with custom story layouts.